From 16f5886e78c856b71d2f29440519ed916863a00b Mon Sep 17 00:00:00 2001
From: Etienne Carriere <etienne.carriere@foss.st.com>
Date: Fri, 2 Feb 2024 17:14:25 +0100
Subject: [PATCH] arm: dts: st: upgrade stm32mp15 Arm SMC watchdog function ID

Upgrade Arm SMC watchdog function ID to the ID merged at upstream
in OP-TEE stm32mp1 platform for watchdog services.

Change-Id: Ia3222b23a0870fcd3a357235903220f173dc631b
Link: https://github.com/OP-TEE/optee_os/commit/e1bfa2fd6db43854eb259c0b8e8e2f1977f9b141
Signed-off-by: Etienne Carriere <etienne.carriere@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/356919
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Yann GAUTIER <yann.gautier@foss.st.com>
---
 arch/arm/boot/dts/st/stm32mp151.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/arch/arm/boot/dts/st/stm32mp151.dtsi
+++ b/arch/arm/boot/dts/st/stm32mp151.dtsi
@@ -42,7 +42,7 @@
 
 	arm_wdt: watchdog {
 		compatible = "arm,smc-wdt";
-		arm,smc-id = <0xb200005a>;
+		arm,smc-id = <0xbc000000>;
 		status = "disabled";
 	};
 
